internal/pkg/scaffold/olm-catalog/csv_updaters.go

```diff
@@ -244,6 +246,20 @@ type CustomResourceDefinitionsUpdate struct {
 	crIDs map[string]struct{}
 }
 
+type descSorter []olmapiv1alpha1.CRDDescription
```

Comment:
Nit: do we need a test for this type? Alternatively, is there a larger test for the `CustomResourceDefinitionsUpdate.Apply` method that we could update?

Reply:
There are no unit tests for `CSVUpdater`s. There really should be :grimacing:. I'll write tests for them, starting with `CustomResourceDefinitionsUpdate.Apply` in a follow-up PR.